docs/source/example_toy_npe_model.md

```diff
@@ -0,0 +1,127 @@
+and unsupported by LALSimulation when `f_max=1024`. If this occurs, it is
+advisable to increase `f_max` to 2048 Hz.
+
+The `waveform_generator` section specifies the `approximant` attribute.
+At present any waveform model, aka `approximant`, that is callable through LALSimulation's
+`SimInspiralFD` API can be used to generate waveforms for dingo via the
+{py:class}`dingo.gw.waveform_generator.waveform_generator.WaveformGenerator` module (see
+[generating_waveforms](generating_waveforms.md)).
+
+The `intrinsic_prior` section is based on Bilby's prior module.
+Default values can be found in `dingo.gw.prior`.
+Two priors to note are the `chirp_mass` and `mass_ratio`, whose minimum values are set
+to 15.0 and 0.125, respectively. Extending these priors towards lower chirp masses
+or more extreme mass-ratios may lead to poor performance of the embedding network
+during training and would require changes to the network setup.
+Note that the `luminosity_distance` and `geocent_time` are defined as constants
+to generate the waveform at a fixed reference point.
+
+The compression section can be set to None for testing purposes. For a practical
+example of how it is used, see the next tutorial.
+
+Step 2 Generating the Amplitude Spectral Density (ASD) dataset
+--------------------------------------------------------------
+
+To generate an ASD dataset run
+
+```
+dingo_generate_asd_dataset --settings_file examples/toy_npe_model/asd_dataset_settings.yaml --data_dir $TRAIN_DIR/asd_dataset_folder
+```
+
+This command will generate an {py:class}`dingo.gw.noise.asd_dataset.ASDDataset` object in the form of an .hdf5 file, which will be used later for training. The reason for specifying a folder instead of a file, as in the waveform dataset example, is because some temporary data is downloaded to create Welch estimates of the ASD. This data can be removed later, but it is sometimes useful for understanding how the ASDs were estimated.
+
+The `examples/toy_npe_model/asd_dataset_settings.yaml` file includes several attributes. `f_s` is the sampling frequency in Hz, `time_psd` is the length of time used for an ASD estimate, and `T` is the duration of each ASD segment. Thus, the value of `time_psd`/`T` gives the number of segments analyzed to estimate one ASD. To avoid spectral leakage, a window is applied to each segment. We use the standard window used in LVK analyses, a Tukey window with a roll off of $\alpha=0.4$. The next attribute, `num_psds_max=1`, defines the number of ASDs stored in the ASD dataset. For now, we will use only one. See the next [tutorial](example_npe_model.md) for a more advanced setup.
+
+Step 3 Training the network
+---------------------------
+
+To train the network run
+
+```
+sed -i 's+/path/to/waveform_dataset.hdf5+'"$TRAIN_DIR"'/waveform_dataset.hdf5+g' examples/toy_npe_model/train_settings.yaml
+sed -i 's+/path/to/asd_dataset.hdf5+'"$TRAIN_DIR"'/asd_dataset_folder/asds_O1.hdf5+g' examples/toy_npe_model/train_settings.yaml
```

Comment:
We could assume already in the YAML file the simple directory structure described above (using the `training/` directory) for model files. This way we can avoid asking the user to edit the files in this way.